var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -860,6 +857,28 @@ def configure_args(self):
 
         return options
 
+    # Copy nvptx-tools into the GCC install prefix
+    def copy_nvptx_tools(self):
+        nvptx_tools_bin_path = join_path(self.spec["nvptx-tools"].prefix, "bin")
```

Comment:
Technically you don't need a `join_path` here.  You can instead:

```suggestion
        nvptx_tools_bin_path = self.spec["nvptx-tools"].prefix.bin
```